lib/rspec_api_docs/formatter/resource/example.rb

```diff
@@ -126,6 +126,8 @@ def request_path(request)
       end
 
       def request_body(body)
+        return nil if body.nil?
```

Comment:
👆 this is the actual fix!

Reply:
This changed in Rack 3.1...
v3.0.x returns a `StringIO` instance, whereas v3.1 returns `nil`!